src/duke/command/Parser.java

```diff
@@ -0,0 +1,261 @@
+package duke.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.exception.InvalidIndexNumberException;
+import duke.exception.InvalidUserCommandException;
+import duke.exception.StorageException;
+import duke.exception.TaskDoesNotExistException;
+import duke.storage.Storage;
+import duke.storage.TaskList;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import duke.ui.Ui;
+
```

Comment:
Class Script is very well documented good job for the extensive documentation.

However Have some concerns regarding the addition of static modifier to many class methods